docs/pluggable_modules.md

```diff
@@ -1,49 +1,35 @@
 # Pluggable modules guide:
 
-Pluggable modules **it's predefined modules**, that you can **only** *enable/disable* by changing config file.
-After updating this config file with enabled addons you need to restart sc4s.
-
-This config file it's a yaml file with list of addons ([whole list of addons here](https://github.com/splunk/splunk-connect-for-syslog/blob/main/package/lite/etc/config.yaml)):
-```
----
-addons:
-    - cisco
-    - paloalto
-    - dell
-```
-
-You don't need to rebuild docker image, you **need mount custom config** into */etc/syslog-ng/config.yaml*.
+Splunk Lite pluggable modules are predefined modules that you can enable or disable by modifying the config.yaml file. This yaml file contains a list of add-ons. See the ([config.yaml reference file](https://github.com/splunk/splunk-connect-for-syslog/blob/main/package/lite/etc/config.yaml)) for more information about this file. Once you update config.yaml you save it to /etc/syslog-ng/config.yaml.
```

Comment:
Can we fix few things? After refactoring sense little-bit loosed.

First thing:

> Splunk Lite pluggable modules are predefined modules that you can enable or disable by modifying the config.yaml file.

>  **SC4S Lite** pluggable modules are predefined modules that you can enable or disable **by the your own config file**.

Second thing:

> See the ([config.yaml reference file](https://github.com/splunk/splunk-connect-for-syslog/blob/main/package/lite/etc/config.yaml)) for more information about this file.

> See the **example** of [config.yaml **(with list of all available pluggable modules)**](https://github.com/splunk/splunk-connect-for-syslog/blob/main/package/lite/etc/config.yaml) for more information about this file.

Third thing:

>  Once you update config.yaml you save it to /etc/syslog-ng/config.yaml

>  Once you update config.yaml you **need mount it to docker container and override** /etc/syslog-ng/config.yaml

Reply:
@jenworthington